sim/visuals/microbit.ts

```diff
@@ -2552,152 +2566,211 @@ namespace pxsim.visuals {
             }, false);
         }
 
-        private attachPinsIOEvents() {
-            this.pins.slice(0, 4).forEach((pin, index) => {
-                // var index = i + 2;
-                if (!this.board.edgeConnectorState.pins[index]) return;
+        private attachPinsTouchEvents() {
+            const pins = this.pins.slice(0, 5);
```

Comment:
Why is this slice necessary?

Reply:
the board has more pins than are touch-able (much like the edge connector on the micro:bit)